src/modules/linkdrop-p2p/index.ts

```diff
@@ -42,6 +46,14 @@ class LinkdropP2P implements ILinkdropP2P {
     this.baseUrl = baseUrl
   }
 
+  parseVersion: TParseVersion = (claimUrl) => {
+    const hashIndex = claimUrl.indexOf('#');
+    const paramsString = claimUrl.substring(hashIndex + 1).split('?')[1]
+    const parsedParams = parseQueryParams(paramsString)
+
+    return  parsedParams["v"] || '1'
```

Comment:
instead of setting default version to '1', raise an error "Version parameter 'v' not found in URL"